src/bp-activity/bp-activity-functions.php

```diff
@@ -1737,6 +2148,33 @@ function bp_activity_format_activity_action_activity_comment( $action, $activity
 	return apply_filters( 'bp_activity_comment_action', $action, $activity );
 }
 
+/**
+ * Format 'activity_like' activity actions.
+ *
+ * @since 14.0.0
+ *
+ * @param string $action   Static activity action.
+ * @param object $activity Activity data object.
```

Comment:
```suggestion
 * @param BP_Activity_Activity $activity Activity data object.
```

Reply:
As it is used to inside the function to fetch a list of activities being regular objects, I'll leave it this way for now. See https://github.com/buddypress/buddypress/blob/54d3c6140bf0a773f06032f8bf9eb112404d3fe7/src/bp-activity/classes/class-bp-activity-activity.php#L836